container.go

```diff
@@ -262,6 +275,7 @@ type Config struct {
 	MemorySwap        int64               `json:"MemorySwap,omitempty" yaml:"MemorySwap,omitempty"`
 	MemoryReservation int64               `json:"MemoryReservation,omitempty" yaml:"MemoryReservation,omitempty"`
 	KernelMemory      int64               `json:"KernelMemory,omitempty" yaml:"KernelMemory,omitempty"`
+	PidsLimit         int64               `json:"PidsLimit,omitempty" yaml:"PidsLimit,omitempty"`
```

Comment:
Maybe the [container inspecting](https://github.com/fsouza/go-dockerclient/blob/bf97c77db7c945cbcdbf09d56c6f87a66f54537b/container_test.go#L183)?